src/test/HostActivationTests/StandaloneAppActivation.cs

```diff
@@ -341,6 +347,10 @@ private IntPtr WaitForPopupFromProcess(Process process, int timeout = 5000)
                 timeout -= 100;
             }
 
+            Assert.True(
```

Comment:
When I've had tests that seemed to be timing out, but I didn't know whether they were hung or just needed more timeout, I used the following scheme
1. If test completes before X seconds, mark it as pass. Otherwise set a flag.
2. If test completes before X+N seconds, fail, logging X+N
3. If test has not completed after X+N seconds, fail, logging that it's hung.
Eg., I might use 30 seconds for X and 90 seconds for N.
Then next time it fails I can see whether it might need a longer X, or it's actually likely hung.